src/spec-lang/typeString_parser.ts

```diff
@@ -0,0 +1,5424 @@
+// Need the ts-nocheck to suppress the noUnusedLocals errors in the generated parser
```

Comment:
If this file is generated, then should we put it to `.gitignore` too?

Reply:
Fixed in 6b91a1f